schemas/wf_run.proto

```diff
@@ -12,26 +12,22 @@ option java_package = "io.littlehorse.sdk.common.proto";
 option csharp_namespace = "LittleHorse.Common.Proto";
 
 message WfRun {
-  reserved 5;
-
   string id = 1;
   string wf_spec_name = 2;
   int32 wf_spec_version = 3;
   LHStatus status = 4;
 
+  // Introduced now since with ThreadRun-level retention, we can't rely upon
+  // thread_runs.size() to determine the number of ThreadRuns.
+  int32 greatest_threadrun_number = 5;
+
   google.protobuf.Timestamp start_time = 6;
   optional google.protobuf.Timestamp end_time = 7;
   repeated ThreadRun thread_runs = 8;
 
   repeated PendingInterrupt pending_interrupts = 9;
   repeated PendingFailureHandler pending_failures = 10;
-
-  // Introduced now since with ThreadRun-level retention, we can't rely upon
-  // thread_runs.size() to determine the number of ThreadRuns.
-  int32 greatest_threadrun_number = 11;
-
-  // Keeps track of old versions of the WfSpec for this WfRun
-  repeated int32 old_wf_spec_versions = 12;
+  repeated WfSpecId old_wf_spec_versions = 11;
```

Comment:
order matters here?

Reply:
Yes, order matters, if we migrate back. This field will serve as a log of what versions we were on, etc.